src/telliot/contract/contracts_dict.py

```diff
@@ -0,0 +1,16 @@
+"""
+A wrapper for accessing all TellorX contracts in one place.
+Each TellorX contract is accessible by its name.
+"""
+from dataclasses import dataclass
+
+from telliot.contract.contract import Contract
+
+
+@dataclass
+class ContractsDict:
```

Comment:
Don't think it should be called `Dict`, since that's a built-in python type. Maybe just `Contracts`?

Reply:
indeed. I renamed to TellorContracts